### PR TITLE
Overloaded get_songs and added tests 

### PR DESCRIPTION
**Pull request by jakeHebert**
I overloaded get_songs() to allow users to return an array of songs for a specific album instead of all songs across their discography. I also updated the test file to test the overloaded function.

### CODE REVIEW

**Comment by jakeHebert** [comment]
The test seems to be failing because python 3.5 couldn't install for the Travis CI build...

File: darklyrics/darklyrics.py [hunk 1]
```diff
@@ -70,18 +70,38 @@ def get_lyrics(song, artist=''):
...
-    result = []
-    for link in links:
-        if 'html#' in link.attrs['href']:
-            result.append(link.text)
-    return result
+def get_songs(artist, album=None):
+    """Will allow a user to return an array of all songs from specific album."""
+    # This will still cuase some problems if a band has multiple albums with the same substring.
+    if album is not None:
+        url = get_artist_url(artist)
```
Comment (res0nance) [inline darklyrics/darklyrics.py]:
```
url = get_artist_url(artist)
        response = requests.get(url)
        soup = BeautifulSoup(response.content, 'html.parser')
        if 'not Found' in soup.title.string:
            raise LyricsNotFound()
```
This bit is repeated try moving it before the if

**Review by res0nance — CHANGES_REQUESTED** [review]

File: darklyrics/darklyrics.py (hunk 1 above)
Comment (jakeHebert) [reply darklyrics/darklyrics.py]: Good catch! Gonna fix that bit.

File: darklyrics/darklyrics.py [hunk 2]
```diff
@@ -70,18 +70,35 @@ def get_lyrics(song, artist=''):
...
-    return result
+    if album is not None:
+        if 'not Found' in soup.title.string:
+            raise LyricsNotFound()
+        album_list = soup.find_all("div", class_="album")
+        for a in album_list:
+            stew_str = str(a.strong)
+            if stew_str.find(str(album)) != -1:
+                soup = BeautifulSoup(str(a), 'html.parser')
+        result = []
```
Comment (res0nance) [inline darklyrics/darklyrics.py]: These bunch of lines are repeated as well

**Review by res0nance — CHANGES_REQUESTED** [review]

**Review by res0nance — APPROVED** [review]
👍 Nice and clean

**Pull request merged**